openwisp_radius/api/views.py

```diff
@@ -341,45 +341,24 @@ def validate_membership(self, user):
             raise serializers.ValidationError({'non_field_errors': [message]})
 
 
-class BatchView(DispatchOrgMixin, generics.CreateAPIView):
+class BatchView(generics.CreateAPIView):
     authentication_classes = (BearerAuthentication,)
```

Comment:
let's add `SessionAuthentication` it makes it easier to test (it's safe to add this authentication class to all endpoints which make use of `BearerAuthentication`.

I tried this, the Browsable API POST form shows up, and I noticed two problems:

- created and modified are shown in the POST form, but these fields should not be displayed (should be readonly)
- I tried posting some data and I got this:

```json
{
    "organization": [
        "Object with slug=342ad600-3ec7-46f2-9e7d-9af1dbd93205 does not exist."
    ]
}
```

Reply:
I was using `curl` to make requests, the form showed some more errors, fixed them all.